test/intl402/NumberFormat/numbering-system-options.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2012 Mozilla Corporation. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-initializenumberformat
+description: >
+    Tests that the options numberingSystem and calendar can be  set through
+    either the locale or the options.
+author: Norbert Lindenberg, Daniel Ehrenberg
+includes: [testIntl.js]
```

Comment:
This PR need to tag with features:  ID since v8 is implementing this behind flag.